utils/src/bulk-insert-patients.ts

```diff
@@ -0,0 +1,142 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+
+import { MetriportMedicalApi, PatientCreate, USState } from "@metriport/api";
+import csv from "csv-parser";
+import fs from "fs";
+
+const apiKey = getEnvVarOrFail("API_KEY");
+const facilityId = getEnvVarOrFail("FACILITY_ID");
+const apiUrl = getEnvVarOrFail("API_URL");
+
+const metriportAPI = new MetriportMedicalApi(apiKey, {
+  baseAddress: apiUrl,
+});
+
+async function main() {
+  const results: PatientCreate[] = [];
+  const errors: Array<{ firstName: string; lastName: string; dob: string; message: string }> = [];
+
+  fs.createReadStream("../insert-patients.csv")
```

Comment:
These are patients related to one customer only, right? Worth adding a note making this clear, otherwise someone could run the script with patients from multiple customers using the same `facilityId`